yt/frontends/flash/tests/test_outputs.py

```diff
@@ -54,6 +57,23 @@ def test_units_override():
     units_override_check(sloshing)
 
 
+@disable_dataset_cache
```

Comment:
I didn't know about this decorator but after reading it, it seems to me that it's currently broken and that the previous state is *never* restored (see `disable_dataset_cache` therein). I've opened #3796 to fix this.